### PR TITLE
Refactored topic prefix to Topic class

### LINKED ISSUE #15: Role name method in Topic model
Topics are stored in the server's role list in the format `Topic | <Topic.name>`. This `Topic | ` prefix has to be prepended (`PREFIX + Topic.getName()`) everywhere that mentorbot interacts with the server's role list. It would be better to create a new method, `Topic.getRoleName()`, that handles this within the model itself, so that the controller layer does not have to worry about this concatenation.

### PR DESCRIPTION
**Pull request by skyegallup**
Closes #15

### CODE REVIEW

**Review by Madrugaur — APPROVED** [review]

**Comment by skyegallup** [comment]
Git history is a little weird now due to backmerge, but everything should be fine here. @peterkos can you give final review?

**Review by Madrugaur — APPROVED** [review]
Looks good

File: src/main/java/entities/Server.java [hunk 1]
```diff
@@ -25,8 +23,11 @@ public Server(Guild guild) {
         List<Role> roles = guild.getRoles();
         for (Role role : roles) {
             String name = role.getName();
-            if (name.startsWith(TOPIC_PREFIX)) {
-                topics.put(name.substring(8), new Topic(name.substring(8), role));
+            if (name.startsWith(Topic.PREFIX)) {
+                topics.put(
+                    name.substring(Topic.PREFIX.length()),
```
Comment (peterkos) [inline src/main/java/entities/Server.java:28]: Is this including the ` | ` in the topic data?

**Review by peterkos — APPROVED** [review]
nice work!

**Pull request merged**